logging.yaml

```diff
@@ -0,0 +1,18 @@
+version: 1
+formatters:
+  simple:
+    format: "%(message)s"
```

Comment:
As indicated above we could also include the module/file here where the logs stem from, thats quite standard in a lot of loggers

Reply:
👍